sdk/documentintelligence/ai-document-intelligence-rest/README.md

```diff
@@ -267,6 +313,86 @@ console.log(response);
 //  }
 ```
 
+## Get analyze PDF
+
+```ts
+const filePath = path.join(ASSET_PATH, "layout-pageobject.pdf");
+
+const base64Source = fs.readFileSync(filePath, { encoding: "base64" });
+
+const initialResponse = await client
+  .path("/documentModels/{modelId}:analyze", "prebuilt-read")
+  .post({
+    contentType: "application/json",
+    body: {
+      base64Source,
+    },
+    queryParameters: { output: ["pdf"] },
+  });
+
+if (isUnexpected(initialResponse)) {
+  throw initialResponse.body.error;
+}
+
+const poller = await getLongRunningPoller(client, initialResponse, { ...testPollingOptions });
+
+await poller.pollUntilDone();
+
+const output = await client
+  .path(
+    "/documentModels/{modelId}/analyzeResults/{resultId}/pdf",
+    "prebuilt-read",
+    parseResultIdFromResponse(initialResponse)
+  )
+  .get();
+
+// A PDF's header is expected to be: %PDF-
+assert.ok(output.body.toString().startsWith("%PDF-"));
```

Comment:
We shouldn't assert in the README snippets. Could we show an example of streaming the content to disk?
```suggestion
```